docs/en/compute/Presto-Iceberg.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: global
+title: Querying Iceberg Tables with Presto and Alluxio
+nickname: Presto-Iceberg
+group: Compute Integrations
+priority: 2
+---
+
+Presto has introduced support for [Iceberg tables](https://iceberg.apache.org/) in version 0.256.
+
+This document describes how to use Presto to query Iceberg tables through Alluxio.
+
+* Table of Contents
+{:toc}
+
+In order to use Presto to query an Iceberg table, make sure you have set up Presto, Hive metastore 
+and Alluxio. If you haven't done so, please refer to the 
+[guide]({{ '/en/compute/Presto.html' | relativize_url }}) on general Presto installation 
+and configuration. Most of that guide apply for Iceberg workflows as well, and this document 
+covers the specific instructions for working with Iceberg tables.
+
+Before proceeding, make sure you have a working setup of Presto, Hive metastore and Alluxio, 
+and Presto can access data through Alluxio's filesystem interface. 
+The [guide]({{ '/en/compute/Presto.html' | relativize_url }}) mentions two approaches that 
+Presto can access data through Alluxio: via the filesystem interface, and via the catalog service. 
+Alluxio has not added support for Iceberg in the catalog service yet, so for Iceberg tables Presto 
```

Comment:
Might be out of scope of this doc, I'm just curious what kind of support needed for Iceberg in catalog service? Did we have any relevant discussions before？

Reply:
I'm not particularly sure about this, but from what I know about the existing Hive support, we need to make SDS understand the Iceberg format so that SDS can mount the correct files, report statistics of the tables to Presto when it's generating query plans, etc.

Reply:
Gotcha, sounds good to me.